vavr/src/main/java/io/vavr/control/Validation.java

```diff
@@ -704,22 +715,26 @@ public String toString() {
     /**
      * An invalid Validation
      *
-     * @param <E> type of the error of this Validation
+     * @param <E> type of the errors of this Validation
      * @param <T> type of the value of this Validation
      */
     final class Invalid<E, T> implements Validation<E, T>, Serializable {
 
         private static final long serialVersionUID = 1L;
 
-        private final E error;
+        private final Seq<? extends E> errors;
```

Comment:
I think it can be `Seq<E> errors`. Maybe it needs to be casted when assigned/at creation time in one of our factory methods.